pkg/features/gardener_features.go

```diff
@@ -30,6 +30,11 @@ const (
 	// owner: @mvladev
 	// alpha: v5.0
 	CoreDNS utilfeature.Feature = "CoreDNS"
+
+	// Logging enables logging stack for clusters.
+	// owner @mvladev,@vlpanov
+	// alpha: v7.0
```

Comment:
what does this version mean?

Reply:
I don't know. @mvladev can you elaborate, please?

Reply:
Gardener version. 

https://github.com/kubernetes/kubernetes/blob/57bb26911d0df6b72ab0bf4161c38649f2aa90e1/pkg/features/kube_features.go#L36-L39

Reply:
Its `alpha` from version `X.0`
it might be `beta` from version `Y.0` onward
and might graduate to `stable` in `Z.0` onward